packages/amplication-build-manager-service/src/app.controller.ts

```diff
@@ -43,7 +43,11 @@ export class AppController {
     try {
       const path = await this.buildContextStorage.saveBuildContextSource(gr);
       const runResponse = await this.buildService.runBuild(path);
-      this.emitInitMessage(runResponse.runId, 'Generating code');
+      this.emitInitMessage(
+        gr.buildId,
```

Comment:
I think `gr` can be `generatedResource`. Let's not be cryptic.

Reply:
Done